tests/bud/no-history/Dockerfile

```diff
@@ -2,6 +2,7 @@
 # entries, but it does have at least one layer.
 
 FROM nixery.dev/shell AS first-stage
```

Comment:
Can we get rid of `nixery` here?

Reply:
Any preference on what to change it to?

Reply:
Okay, `example.com` unless/until I hear otherwise.